README.md

```diff
@@ -1 +1,86 @@
-# iagora_deployment
\ No newline at end of file
+# iagora_deployment
+
+
+
+## How to use vm bicep deployment templates
+
+To deploy the virtual machine and associated resources using the provided Bicep template, follow these steps:
+
+### Step 1: Install Azure CLI
+
+Ensure that the Azure CLI is installed. If it is not installed, use the following command:
+
+```bash
+curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+```
+
+### Step 2: Log in to Your Azure Account
+
+After installing Azure CLI, log in to the Azure account:
+
+```bash
+az login
+```
+
+This command will open a browser window to log in with Azure credentials. If using a remote machine without a graphical interface, use:
+
+```bash
+az login --use-device-code
+```
+
+### Step 3: Install Bicep CLI
+
+Once logged in, install Bicep CLI by running:
+
+```bash
+az bicep install
+```
+
+### Step 4: Upgrade Bicep CLI (Optional)
+
+To ensure the latest version of Bicep is installed, upgrade it using:
+
+```bash
+az bicep upgrade
+```
+
+### Step 5: Verify Bicep Installation
+
+Verify the Bicep installation by checking the version:
+
+```bash
+az bicep version
+```
+
+### Step 6: (Optional) Convert Existing ARM Template to Bicep
+
+To convert an ARM template in JSON format to Bicep, use:
+
+```bash
+az bicep decompile -f path/to/ARM/template.json
+```
```

Comment:
```suggestion
```
We don't provide ARM templates yet, right?

Reply:
right !